src/js/components/modals/AppModalComponent.jsx

```diff
@@ -209,6 +243,7 @@ var AppModalComponent = React.createClass({
 
     var appConfigProps = {
       app: state.app,
+      features: state.info.features,
```

Comment:
Apparently `features` is a part of `marathon_config` and not a top level member. Could you please adjust this line?